src/main/java/lotto/domain/Lottos.java

```diff
@@ -0,0 +1,27 @@
+package lotto.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import lotto.global.LottoRank;
+
+public class Lottos {
+
+    private final List<Lotto> lottos;
+
+    public Lottos(List<Lotto> lottos) {
+        this.lottos = lottos;
+    }
+
+    public List<LottoRank> getWinningLottos(List<Integer> winningNumbers, int bonusNum) {
+        return lottos.stream()
+                .map(lotto -> lotto.checkWinningStatus(winningNumbers, bonusNum))
+                .filter(Objects::nonNull)
+                .collect(Collectors.toList());
```

Comment:
```suggestion
                .toList();
```

중요한건 아니지만 .collect(Collectors.toList()) 대신 .toList()만 사용하셔도 동일한 결과를 얻을 수 있습니다.